lib/RESTClient.js

```diff
@@ -393,6 +395,53 @@ class RESTClient {
             callback(errors.InternalError);
         }).end();
     }
+
+    /**
+    *   send a request to get all raft sessions
+    *   get server's response and return it
+    *
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
+    *   @param {werelogs.Logger} [reqLogger] - Logger instance
+    *   @return {undefined}
+    */
+    getAllRafts(reqUids, callback, reqLogger) {
+        const log = reqLogger || this.createLogger(reqUids);
+        const path = '/_/raft_sessions/';
+        log.debug('getting all raftSessions');
+        this.request('GET', path, log, null, null, callback);
+    }
+
+    /**
+    *   Get raft logs from bucketd
+    *   get server's response and return it
+    *
+    *   @param {string} raftId - raft session id
+    *   @param {number} [start=1] - starting sequence number
+    *   @param {number} [end=10000] - ending sequence number
```

Comment:
According to Jonathan's request, we could rename it `limit`, set the default to `undefined`, and change the description into something that tells that we're setting a limit in terms of number of entries returned (10k being a harcoded max).